Breadcrumbs.py

```diff
@@ -108,4 +132,213 @@ def get_points_by_row(row):
               breadcrumbs[index_of_breadcrumb] = breadcrumbs[index_of_breadcrumb][0:length_of_trim]
           break
       number_of_characters_left -= current_length
-    view.set_status('breadcrumbs',separator.join(breadcrumbs+['']))
+  return breadcrumbs
+
+
+def copy(view, text):
+  sublime.set_clipboard(text)
+  sublime.status_message('Breadcrumbs copied to clipboard')
+
+
+class BreadcrumbsEventListener(sublime_plugin.ViewEventListener):
```

Comment:
Oooooooo, I really like the idea of the phantoms sticking to the top of the screen. Not sure that’ll work, but since you’re really pointing to information that’s off-screen it’s totally the right spot to do so. 

It’s with things like this that you appreciate the total freedom to create UI in something like Atom.